aplib.net-demo/Assets/Scripts/Points/ItemPointsAdder.cs

```diff
@@ -0,0 +1,17 @@
+using Assets.Scripts.Items;
+using UnityEngine;
+
+public class ItemPointsAdder : PointsAdderComponent
+{
+    private void Awake()
+    {
+        PickupableItem item = GetComponent<PickupableItem>();
+        item.ItemPickedUp += SendPoints;
+    }
```

Comment:
You should be subscribing to and unsubscribing from events in OnEnable() and OnDisable.